addons/web/static/src/js/services/crash_manager.js

```diff
@@ -122,6 +123,19 @@ var CrashManager = AbstractService.extend({
                     window.onOriginError();
                     delete window.onOriginError;
                 } else {
+                    // In Safari 16.4+ (as of Jun 14th 2023), an error occurs
+                    // when going back and forward through the browser when the
+                    // cache is enabled. A feedback has been reported but in the
+                    // meantime, hide any script error in these versions.
+                    if (
+                        /^((?!chrome|android).)*safari/i.test(navigator.userAgent)
+                        && ("maxTouchPoints" in navigator)
```

Comment:
:warning: Don't forget that *every browser* on iOS uses WKWebView as rendering engine, thus all are impacted in some way by this issue (tested at least with Firefox for iOS). So, I wouldn't filter too much...

BTW, `web.config` already implements a `isIOS` flag for that purpose.